kernel/packages/decentraland-ecs/src/decentraland/Events.ts

```diff
@@ -48,7 +48,7 @@ function createSubscriber(eventName: keyof IEvents) {
  * This event is triggered when you change your camera between 1st and 3rd person
  * @public
  */
-export const onCameraModeChanged = new Observable<IEvents['onCameraModeChanged']>(createSubscriber('onCameraModeChanged'))
+export const onCameraModeChanged = new Observable<IEvents['cameraModeChanged']>(createSubscriber('cameraModeChanged'))
```

Comment:
```suggestion
export const cameraModeChanged = new Observable<IEvents['cameraModeChanged']>(createSubscriber('cameraModeChanged'))
```
If we are going to change the event name, we shouldn't do it for this name as well?

Reply:
I see that the enter and leave scenes have the `on` moniker too. Maybe we should rename them too?

Reply:
Yes, and there are more IEvents with `on`
I talked with @menduz and that code it's already in production, we cannot change it.

`cameraModeChange` is not in production yet.

Reply:
There is a pending change about this: decentraland/unity-renderer#295